sdk/github.md

```diff
@@ -205,8 +205,24 @@ tied in with the [SDK Upload Action](https://github.com/ably/sdk-upload-action).
 
 ## Labels
 
-We are slowly converging on a set of labels that are useful for our context and these are being duplicated across client libraries when needed.
-This list is canonically defined [here](https://github.com/ably/ably-common/blob/main/github/labels.yml).
+The following table canonically defines labels we use in common across our open source SDK repositories:
+
+| Name | Color | Description | Additional Notes |
+| ---- | ----------- | ----- | ---------------- |
+| <span style="padding: 0.25em 0.75em; border-radius: 1em; font-weight: 500; white-space: nowrap; background-color: #fbca04; color: black;">blocked-by-ably</span> | `fbca04` | We can't proceed until something under our direct control, in a different codebase, happens. | Comments should be added to indicate what the blockage is (e.g. another SDK repository). |
+| <span style="padding: 0.25em 0.75em; border-radius: 1em; font-weight: 500; white-space: nowrap; background-color: #fbca04; color: black;">blocked-by-external</span> | `fbca04` | We can't proceed until something outside of Ably's direct control happens. | Comments should be added to indicate what the blockage is. |
+| <span style="padding: 0.25em 0.75em; border-radius: 1em; font-weight: 500; white-space: nowrap; background-color: #d73a4a; color: white;">bug</span> | `d73a4a` | Something isn't working. It's clear that this does need to be fixed. | Usually implies that related changes can be released in a `patch` version bump. |
+| <span style="padding: 0.25em 0.75em; border-radius: 1em; font-weight: 500; white-space: nowrap; background-color: #ec4b42; color: white;">breaking</span> | `ec4b42` | Affects the developer experience when working in our codebase. | Implies a need to release related changes in a `major` version bump. |
+| <span style="padding: 0.25em 0.75em; border-radius: 1em; font-weight: 500; white-space: nowrap; background-color: #ff88cc; color: black;">code-quality</span> | `ff88cc` | Affects the developer experience when working in our codebase. |
+| <span style="padding: 0.25em 0.75em; border-radius: 1em; font-weight: 500; white-space: nowrap; background-color: #0075ca; color: white;">documentation</span> | `0075ca` | Improvements or additions to public interface documentation (API reference or readme). |
+| <span style="padding: 0.25em 0.75em; border-radius: 1em; font-weight: 500; white-space: nowrap; background-color: #a2eeef; color: black;">enhancement</span> | `a2eeef` | New feature or request. | Implies a need to release related changes in a `minor` version bump. |
```

Comment:
Some features create breaking changes so should we amend the phrasing so that it says "Implies a need to release related changes in a 'minor' or 'major' version bump?

Reply:
Thanks, this is a valid observation. If a feature is breaking then my expectation is that it would have _both_ the `enhancement` and `breaking` labels... this is something I'm expanding on for my next pull request which enhances the guidance around releases, but in the meantime I think I can add some explanatory wording to this table on this point. I'll push up a commit shortly to do that...

Reply:
Wording added: https://github.com/ably/engineering/pull/44/commits/b3cc90f638f0795ecec3a0cfe4d02b3118f68029